pkg/node/node.go

```diff
@@ -698,6 +701,15 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 func (b *Bee) Shutdown(ctx context.Context) error {
 	var mErr error
 
+	// if a shutdown is already in process, return here
+	b.shutdownMutex.Lock()
```

Comment:
why not use `sync.Once`?

Reply:
The issue here is the handling in pidKiller. We need to know if the shutdown is already in progress so we don't kill the process. So if the Shutdown call triggered this from the postage listener we don't do anything.

Reply:
you could still use `sync.Once` :)